### DISCUSSION TITLE
I don't see an arrow "use existing note", to append to.

### DISCUSSION THREAD

**Discussion by LightTemplar** [coddingtonbear/obsidian-web#108] [Q&A] [answered]
If I click on Obsidian Web on the page, which is already on Obsidian, I see the message about it, but don't see the arrow "use existing note", as it mentioned here: https://github.com/coddingtonbear/obsidian-web/discussions/82
This is how it looks by me:
![image](https://github.com/coddingtonbear/obsidian-web/assets/453783/effa8633-ee5d-45f0-a4d6-e1b6c3cfb051)

**Comment by LightTemplar** [comment]
@coddingtonbear, why have you moved it into discussions? What is it here to discuss?

**Comment by coddingtonbear** [comment]
Oh, it's just that this is _probably_ just that you haven't set up the machinery supporting this.  It's probable that it's not a bug, but a misconfiguration -- there's no magic here!  Make sure you've configured your "Note Recall" settings correctly -- specifically the control next to the text "When the URL of the page you are visiting has been found in the content of a note in your vault, suggest this template for updating the existing note".

**Comment by LightTemplar** [comment]
That's how my settings look now:
![image](https://github.com/coddingtonbear/obsidian-web/assets/453783/4e0f8035-868b-4ca4-a8f4-741336ff4a26)

![image](https://github.com/coddingtonbear/obsidian-web/assets/453783/30564469-f3bc-4661-826f-3a977db6d15d)


I paid a closer attention on those words "found to match the `url` field in the frontmatter" in my settings. Does it have to be the `Url` field? I use `SourceUrl` in frontmatter. And this is different from what you mention above. I have Plugin ver. 2.2.3 in the Brave Browser (Chrome clone)

**Reply by coddingtonbear** [reply, marked as answer]
Your earlier screenshot is a _mention_ so not showing the arrow is expected because you haven't configured a template to suggest for mentions.

For direct page notes, yes, you must use the `url` field. https://github.com/coddingtonbear/obsidian-web/wiki/Tip:-Special-Frontmatter-Fields#for-finding-notes-for-your-url

**Reply by LightTemplar** [reply]
I added `url` field, and now I have that arrow, but it looks so now. 
![image](https://github.com/coddingtonbear/obsidian-web/assets/453783/adaf13a7-6def-4b2e-a522-635a04a24c13)
What does that orange sign mean? I'm asking, because using that arrow and saving to Obsidian leads to 40400 Error. And maybe there is a connection. I checked relevant discussion https://github.com/coddingtonbear/obsidian-web/discussions/82 , but didn't find anything related.

**Reply by LightTemplar** [reply]
Using of that arrow works now, can't tell what changed. And that orange sign is still also here.

**Reply by coddingtonbear** [reply]
If you see a note like the above, it means that we found either a directly-referenced page (one where the `url` frontmatter field matches) or a mention (one where the URL just happens to appear in one of your notes).  In the above case: it's a directly-referenced page.

Whether that arrow on the right half of that message appears and what it does, though, isn't magic, and is configured via your "Note Recall" settings.  My guess, at a glance, is that the template you've selected in your "Note Recall" settings isn't set up correctly and you'll want to change it such that it does perform the action you're hoping for.  If that isn't super clear, you can always reset your templates to their default settings -- the default templates _do_ work.
